pkg/kubectl/cmd/config/get_contexts.go

```diff
@@ -81,8 +81,10 @@ func NewCmdConfigGetContexts(out io.Writer, configAccess clientcmd.ConfigAccess)
 			cmdutil.CheckErr(options.RunGetContexts())
 		},
 	}
-	cmdutil.AddOutputFlags(cmd)
-	cmdutil.AddNoHeadersFlags(cmd)
+
+	cmd.Flags().Bool("no-headers", false, "When using the default or custom-column output format, don't print headers (default print headers).")
+	cmd.Flags().StringP("output", "o", "", "Output format. One of: json|yaml|wide|name|custom-columns=...|custom-columns-file=...|go-template=...|go-template-file=...|jsonpath=...|jsonpath-file=... See custom columns [http://kubernetes.io/docs/user-guide/kubectl-overview/#custom-columns], golang template [http://golang.org/pkg/text/template/#pkg-overview] and jsonpath template [http://kubernetes.io/docs/user-guide/jsonpath].")
+	cmd.Flags().Bool("allow-missing-template-keys", true, "If true, ignore any errors in templates when a field or map key is missing in the template. Only applies to golang and jsonpath output formats.")
```

Comment:
suspicious, what uses this.

Reply:
The only output formats this command uses are `name` and empty string. I originally added this as it was bound to the command as part of the call to `cmdutil.AddPrinterFlags()`. Will remove